shell/platform/linux/fl_view.cc

```diff
@@ -733,5 +688,5 @@ void fl_view_end_frame(FlView* view) {
 
   gtk_container_forall(GTK_CONTAINER(view), fl_view_reorder_forall, &data);
```

Comment:
This is also `gtk_container_forall(GTK_CONTAINER(self), fl_view_reorder_forall, &data);`

Reply:
Done.